lib/groonga/client/request/select.rb

```diff
@@ -97,6 +97,16 @@ def query(value)
         #     The new request object for setting a filter condition.
         #
         #   @since 0.4.3
+        #
+        #   @example: Use between function
+        #      request.
+        #        filter.between("tags", "min", "min_border", "max", "max_border")
+        #          # -> --filter 'between(tags, min, "min_border", max, "max_border")'
```

Comment:
Example should be "workable". `"min_border"` and `"max_border"` are "invalid" border. It should be `"include"` or `"exclude"`. And `"min"` and `"max"` should be more useful values.